k8s/podman/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM quay.io/podman/stable:v5.0.2
+RUN ln -sfT $(type -p podman) /usr/bin/docker
```

Comment:
why need link? podman provides docker compat interface and has such link (or wrapper) ON, does it?

Reply:
@mike-dubman please see https://github.com/mgoltzsche/podman-static/blob/master/Dockerfile#L147
The maintainer of the image also creates a symbolic link.
Please also check the latest update on the PR - I've switched to the official Podman image from `quay.io` and added symbolic link creation as a pre-step before building an image using podman.